packages/json-schemas/schemas/basic_type_schemas.ts

```diff
@@ -4,6 +4,12 @@ export const addressSchema = {
     pattern: '^0x[0-9a-f]{40}$',
 };
 
+export const hexSchema = {
+    id: '/Hex',
+    type: 'string',
+    pattern: '^0x[0-9a-f]',
```

Comment:
It should be `^0x[0-9a-f]*`.
Let's write tests for the schemas so that we don't have those problems